crates/apub_receive/src/activities/receive/community.rs

```diff
@@ -117,3 +210,23 @@ pub(crate) async fn receive_undo_remove_community(
 
   Ok(())
 }
+
+/// Checks if the remote user is creator of the local community. This can only happen if a community
+/// is created by a local user, and then transferred to a remote user.
+async fn verify_is_remote_community_creator(
+  user: &Person,
+  community: &Community,
+  context: &LemmyContext,
+) -> Result<(), LemmyError> {
+  let community_id = community.id;
+  let community_mods = blocking(context.pool(), move |conn| {
+    CommunityModeratorView::for_community(conn, community_id)
+  })
+  .await??;
+
+  if user.id != community_mods[0].moderator.id {
+    Err(anyhow!("Actor is not community creator").into())
+  } else {
+    Ok(())
+  }
```

Comment:
Eventually we should name this `top_mod` or something since we removed community creator.